dashboard/test/models/levels/bubble_choice_test.rb

```diff
@@ -229,6 +229,9 @@ class BubbleChoiceTest < ActiveSupport::TestCase
       level 'sublevel_3_copy'
     DSL
 
+    # access a translation, to trigger any file reads, before we stub File.read
```

Comment:
might be worth explicitly calling out here that the rails backend lazy-loads translations when a translation is looked up for the first time, and that because that process involves doing a bunch of File.reads, we want to make sure it happens before stubbing